### PR TITLE
Addwithsymbols to applyscheduledupdates

### PR DESCRIPTION
**Pull request by hud10837**
These were pretty straightforward, but I was a bit unsure about animate-3d-graphic! I left out the camera controls because they weren't in common and I thought they might be self evident.

### CODE REVIEW

File: java/animate-3d-graphic/README.md [hunk 1]
```diff
@@ -1,58 +1,52 @@
...
-![Animate 3D Graphic App](animate-3d-graphic.png)
+![Image of animate 3D graphic](animate-3d-graphic.png)
 
 ## How to use the sample
 
 Animation Controls (Top Left Corner):
+
 * Select a mission -- selects a location with a route for plane to fly.
 * Mission progress -- shows how far along the route the plane is. Slide to change keyframe in animation.
 * Play -- toggles playing and stopping the animation.
```
Comment (Rachael-E) [inline java/animate-3d-graphic/README.md]: When the sample is running,the Play button says Stop (changes when you hit it), so let's make that a bit clearer here
```suggestion
* Play/Stop -- toggles playing and stopping the animation.
```

File: java/animate-3d-graphic/README.md [hunk 2]
```diff
@@ -1,58 +1,52 @@
...
 * Follow -- toggles camera following plane.
```
Comment (Rachael-E) [inline java/animate-3d-graphic/README.md]:
```suggestion
* Follow/Free Cam -- toggles camera following plane.
```

File: java/animate-3d-graphic/README.md [hunk 3]
```diff
@@ -1,58 +1,52 @@
...
 
 Speed Slider (Top Right Corner):
+
```
Comment (Rachael-E) [inline java/animate-3d-graphic/README.md]: let's keep this all on one line rather than split it

Comment (hud10837) [reply java/animate-3d-graphic/README.md]: How do you mean? As in
`* Speed Slider (Top Right Corner): Controls speed of animation.` ?

Comment (Rachael-E) [reply java/animate-3d-graphic/README.md]: yeh!

Comment (hud10837) [reply java/animate-3d-graphic/README.md]: Does this not conflict with the same type of descriptor `Animation Controls (Top Left Corner)`? Happy to change it,  but that looks more confusing to me personally

Comment (Rachael-E) [reply java/animate-3d-graphic/README.md]: oh, I see what you mean now, missed that before. Sure, that's fine to leave as is then. Thanks!

**Pull request merged**